requirements-dev.txt

```diff
@@ -6,14 +6,7 @@
 # Since that file does its own testing setup
 
 # Required for environment
-flake8
-autopep8
-flake8-blind-except
-flake8-builtins
-flake8-copyright
-flake8-docstrings
-flake8-logging-format
-flake8-rst-docstrings
```

Comment:
Just `flake8-rst-docstrings` is missing. I figured that since we don't have any `.rst` related errors when linting everything with `ruff` now, maybe that's good enough? I can monitor and see if anything comes up.